src/Node.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Helmich\TypoScriptParser;
```

Comment:
`Helmich\TypoScriptParser\Parser\AST` might be the suitable namespace for this class.